selfdrive/locationd/locationd.py

```diff
@@ -23,8 +23,10 @@
 MAX_FILTER_REWIND_TIME = 0.8  # s
 MAX_SENSOR_TIME_DIFF = 0.1  # s
 YAWRATE_CROSS_ERR_CHECK_FACTOR = 30
-INPUT_INVALID_THRESHOLD = 0.5
-INPUT_INVALID_DECAY = 0.9993  # ~10 secs to resume after a bad input
+INPUT_INVALID_THRESHOLD = 0.5  # 0 bad inputs ignored
+TIMING_INVALID_THRESHOLD = 2.5  # 2 bad timings ignored
+INPUT_INVALID_DECAY = 0.9993  # ~10 secs to resume after exceeding allowed bad inputs by one
+TIMING_INVALID_DECAY = 0.9986  # ~5 secs to resume after exceeding allowed bad timings by one
```

Comment:
think we want this to go in rednose

Reply:
rednose is quite low level. i was thinking about putting it into [pose_kf](https://github.com/commaai/openpilot/blob/master/selfdrive/locationd/models/pose_kf.py) where the filter config is (obs_noise, initial_x, initial_P). but we use the same interface for all the filters here and xx, so i would need to refactor all of them